dfastmi/io/CelerObject.py

```diff
@@ -37,36 +37,52 @@
     CelerProperties
 
 """
-from abc import ABC, abstractmethod
-from typing import List
+from abc import ABC
+from typing import List, Tuple
+from pydantic import BaseModel
+from dfastmi.io.AReach import AReach
 
-
-class ICelerObject(ABC):
+class ICelerObject(ABC, BaseModel):
```

Comment:
I suggest introducing a `DFastBaseModel` that inherits from pydantic's BaseModel to configure this class for all of D-FAST, so that all of our objects such as CelerObject, Branch, etc. can inherit from this and all are configured similarly.

Reply:
Lets have a team meeting/discussion about this and resolve for multiple objects in new issue.

Reply:
I will try and do this. Good suggestion!